src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -735,9 +737,20 @@ bool Dataloader::_load_map_dir(GameManager& game_manager) const {
 		ret = false;
 	}
 
-	if (!map.load_region_file(parse_defines(lookup_file(append_string_views(map_directory, region))).get_file_node())) {
-		Logger::error("Failed to load region file!");
-		ret = false;
+	{
+		std::vector<colour_t> colours;
+		if (!Map::load_region_colours(parse_defines(lookup_file(append_string_views(interface_directory, region_colours))).get_file_node(), colours)) {
+			Logger::error("Failed to load region colours file!");
+			ret = false;
+		}
+
+		using namespace OpenVic::colour_literals;
+		colours.push_back(0xFFFFFF_rgb);
```

Comment:
```suggestion
		colours.push_back(0xFFFFFF_rgb); /* This ensures there is always at least one region colour. */
```
A comment here would be nice, makes the code seem less arbitrary.